From e26941bc8b5a6c197a127879bbfcc5a982c47ee7 Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Fri, 14 Sep 2018 13:37:01 +0200
Subject: [PATCH 0691/1051] Documentation/bindings: phy: update references to
 cp110

The cp110 references in dts has changed, reflect it in comphy
documentation.

Change-Id: I16be0c536d81868265a6c4c87d6abf3048e45007
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 .../devicetree/bindings/phy/phy-mvebu-comphy.txt          | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/Documentation/devicetree/bindings/phy/phy-mvebu-comphy.txt b/Documentation/devicetree/bindings/phy/phy-mvebu-comphy.txt
index c07a75ffd686..25af39d73e1d 100644
--- a/Documentation/devicetree/bindings/phy/phy-mvebu-comphy.txt
+++ b/Documentation/devicetree/bindings/phy/phy-mvebu-comphy.txt
@@ -25,20 +25,20 @@ Required properties (child nodes):
 
 Example:
 
-	cpm_comphy: phy@120000 {
+	CP110_LABEL(comphy): phy@120000 {
 		compatible = "marvell,comphy-cp110";
 		reg = <0x120000 0x6000>;
 		reg-names = "comphy";
-		marvell,system-controller = <&cpm_syscon0>;
+		marvell,system-controller = <&CP110_LABEL(syscon0)>;
 		#address-cells = <1>;
 		#size-cells = <0>;
 
-		cpm_comphy0: phy@0 {
+		CP110_LABEL(comphy0): phy@0 {
 			reg = <0>;
 			#phy-cells = <1>;
 		};
 
-		cpm_comphy1: phy@1 {
+		CP110_LABEL(comphy1): phy@1 {
 			reg = <1>;
 			#phy-cells = <1>;
 		};
-- 
2.17.1

